assets/rpip-10/expense_2.md

```diff
@@ -0,0 +1,18 @@
+This expense has two parts:
+- Transfer the sum of periods 12, 13, 14, 15 in the `IMC amount to make up` column below to the IMC
+- Transfer the sum of periods 12, 13, 14, 15 in the `GMC amount to make up` column below to the GMC
+
+## Rationale
+There was a mismatch between the actual budget (in RPIP) and the intent (in vote text) as explained
+in <https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff>.
+
+This expense compensates the difference during periods affected before the budget was updated.
+
+| Period | RPL to treasury | IMC amount to make up | GMC amount to make up |
+|--------|-----------------|-----------------------|-----------------------|
+| n      | x               | `x*(.34-.27)=.07x`    | `x*(.205-.16)=.045x`  | 
+| 12     | 16123.76        | 1128.67               | 725.57                | 
```

Comment:
If we take the `.07x` seriously in the column heading, note that `07 × 16123.76 = 1128.6632 ≠ 1128.67`.

Also why are we rounding to 2 decimal places anyway?

Reply:
Convenience and readability balanced against relevance. 0.005 RPL rounding error is negligible compared to the budgets in question.

Reply:
OK but we're off by 0.0068 RPL here, which is > 0.005.

Reply:
Ah - got it. You're saying I rounded badly. Fair enough - fixing.